Jenkinsfile

```diff
@@ -1,21 +1,30 @@
 pipeline {
   agent any
+  options {
+    disableConcurrentBuilds()
+    copyArtifactPermission('*')
+  }
   stages {
     stage('workflow:sandbox') {
-      when { anyOf { environment name: 'DATAGOV_WORKFLOW', value: 'sandbox' } }
+      when {
+        allOf {
+          environment name: 'DATAGOV_WORKFLOW', value: 'sandbox'
+          anyOf {
+            branch 'develop'
+          }
+        }
+      }
```

Comment:
Does this stage not require [an `environment` block like the one below](https://github.com/GSA/datagov-deploy/pull/2946/files#diff-e6ffa5dc854b843b3ee3c3c28f8eae2f436c2df2b1ca299cca1fa5982e390cf8R50-R53)?